src/components/TextInput/TextInput.stories.mdx

```diff
@@ -16,7 +16,7 @@ import DSProvider from "../../theme/provider";
   title={getCategory("TextInput")}
   component={TextInput}
   decorators={[withDesign]}
-  parameters={{https://github.com/NYPL/nypl-design-system/pull/1137/conflict?name=src%252Fcomponents%252FTextInput%252FTextInput.stories.mdx&ancestor_oid=4e64f4d5ac327c092247f43cef563cd111b2eba9&base_oid=18657f129e8a84e8b5dba19629daab900adca055&head_oid=f563417ec7a35ada4ac395b9431d54be572f60f7
```

Comment:
I wonder where this came from.

Reply:
Github is telling me it came in 5 months ago from a large code push that Olivia comitted.  An errant paste?  Something weird from a merge?  Not sure.